src/typescript/azure-openapi-validator/rules/RequiredSystemDataInNewApiVersions.ts

```diff
@@ -0,0 +1,64 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+import { nodes, stringify } from "jsonpath"
+import { MergeStates, OpenApiTypes, rules } from "../rule"
+import { getResolvedJson, getResolvedResponseSchema, getResponseSchema } from "./utilities/rules-helper"
+export const RequiredSystemDataInNewApiVersions: string = "RequiredSystemDataInNewApiVersions"
+
+rules.push({
+  id: "R4009",
+  name: RequiredSystemDataInNewApiVersions,
+  severity: "warning",
+  category: "ARMViolation",
+  mergeState: MergeStates.composed,
+  openapiType: OpenApiTypes.arm | OpenApiTypes.dataplane,
+  appliesTo_JsonQuery: "",
+  async *asyncRun(doc, node, path) {
+    if (doc.info) {
+      const apiVersion = doc.info.version
```

Comment:
Do all specs api-version follow this format? I think some RP doesn't follow this format.

Reply:
all management-plane following this format